spec/helpers/helpers_spec.rb

```diff
@@ -1,17 +1,17 @@
 module Helpers
   def login(email, password)
     visit root_path
-    click_link "Войти"
+    click_link I18n.t('.layouts.application.login')
     fill_in :email, with: email
     fill_in :password, with: password
-    click_button "Войти"
+    click_button I18n.t('.user_sessions.new.sign_in')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.